Competition2017/src/competition/subsystems/agitator/commands/SetAgitatorSpeedCommand.java

```diff
@@ -0,0 +1,38 @@
+package competition.subsystems.agitator.commands;
+
+import org.apache.log4j.Logger;
+import com.google.inject.Inject;
+import competition.subsystems.agitator.AgitatorSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.XPropertyManager;
+
+public class SetAgitatorSpeedCommand extends BaseCommand {
```

Comment:
I didn't think we needed velocity speed control for the agitators? But even if we do, we should do this in the same way as the other systems now where there is a stop and a forward command instead of this set speed model?